explorer/src/components/ExplorerStats/ExplorerStats.tsx

```diff
@@ -536,81 +521,17 @@ const ExplorerStats: React.FC<StatsProps> = ({
                     imgStyle: { height: 110 },
                   },
                   {
-                    header: ChainID[3],
-                    src: terraIcon,
-                    to: `${explorer}?emitterChain=3`,
-                    description: (
-                      <ChainOverviewCard
-                        totals={totals}
-                        notionalTransferredToCumulative={
-                          notionalTransferredToCumulative
-                        }
-                        notionalTransferred={notionalTransferred}
-                        dataKey="3"
-                      />
-                    ),
-                    imgStyle: { height: 110 },
-                  },
-                  {
-                    header: ChainID[4],
-                    src: binanceChainIcon,
-                    to: `${explorer}?emitterChain=4`,
-                    description: (
-                      <ChainOverviewCard
-                        totals={totals}
-                        notionalTransferredToCumulative={
-                          notionalTransferredToCumulative
-                        }
-                        notionalTransferred={notionalTransferred}
-                        dataKey="4"
-                      />
-                    ),
-                    imgStyle: { height: 110 },
-                  },
-                  {
-                    header: ChainID[5],
-                    src: polygonIcon,
-                    to: `${explorer}?emitterChain=5`,
-                    description: (
-                      <ChainOverviewCard
-                        totals={totals}
-                        notionalTransferredToCumulative={
-                          notionalTransferredToCumulative
-                        }
-                        notionalTransferred={notionalTransferred}
-                        dataKey="5"
-                      />
-                    ),
-                    imgStyle: { height: 110 },
-                  },
-                  {
-                    header: ChainID[6],
-                    src: avalancheIcon,
-                    to: `${explorer}?emitterChain=6`,
-                    description: (
-                      <ChainOverviewCard
-                        totals={totals}
-                        notionalTransferredToCumulative={
-                          notionalTransferredToCumulative
-                        }
-                        notionalTransferred={notionalTransferred}
-                        dataKey="6"
-                      />
-                    ),
-                    imgStyle: { height: 110 },
-                  },
-                  {
-                    header: ChainID[7],
-                    src: oasisIcon,
-                    to: `${explorer}?emitterChain=7`,
+                    header: ChainID[13],
+                    src: alephiumIcon,
+                    to: `${explorer}?emitterChain=255`,
```

Comment:
It seems this is lost: `imgStyle: { height: 110 }`

Reply:
it is not removed, as we can see [here](https://github.com/alephium/wormhole-fork/pull/40/files/6d6121f084c8585f3b9271159602f286a83f6aaf#diff-1243e033d2e07c5bbbfd4d02c9599bba11f348d05d7d9e37b1169a5941aa15bdR527-R537)